YAXLibTests/ReflectionUtilsTest.cs

```diff
@@ -199,4 +199,25 @@ public void GetSetPropertyValues()
         Assert.That(baseClassProperty1AfterSet, Is.EqualTo(113));
         Assert.That(baseClassProperty2AfterSet, Is.EqualTo(123));
     }
-}
\ No newline at end of file
+
+    [Test]
+    public void GetDefaultValueTest()
+    {
+        Assert.That(ReflectionUtils.GetDefaultValue(typeof(string)), Is.Null);
+        Assert.That(ReflectionUtils.GetDefaultValue(typeof(bool)), Is.False);
+        Assert.That(ReflectionUtils.GetDefaultValue(typeof(char)), Is.EqualTo((char) 0));
+        Assert.That(ReflectionUtils.GetDefaultValue(typeof(sbyte)), Is.EqualTo((sbyte) 0));
+        Assert.That(ReflectionUtils.GetDefaultValue(typeof(byte)), Is.EqualTo((byte) 0));
+        Assert.That(ReflectionUtils.GetDefaultValue(typeof(short)), Is.EqualTo((short) 0));
+        Assert.That(ReflectionUtils.GetDefaultValue(typeof(ushort)), Is.EqualTo((ushort) 0));
+        Assert.That(ReflectionUtils.GetDefaultValue(typeof(int)), Is.EqualTo(0));
+        Assert.That(ReflectionUtils.GetDefaultValue(typeof(uint)), Is.EqualTo(0U));
+        Assert.That(ReflectionUtils.GetDefaultValue(typeof(long)), Is.EqualTo(0L));
+        Assert.That(ReflectionUtils.GetDefaultValue(typeof(ulong)), Is.EqualTo(0UL));
+        Assert.That(ReflectionUtils.GetDefaultValue(typeof(float)), Is.EqualTo(0F));
+        Assert.That(ReflectionUtils.GetDefaultValue(typeof(double)), Is.EqualTo(0D));
+        Assert.That(ReflectionUtils.GetDefaultValue(typeof(decimal)), Is.EqualTo(0M));
+        Assert.That(ReflectionUtils.GetDefaultValue(typeof(DateTime)), Is.EqualTo(DateTime.MinValue));
+        Assert.That(ReflectionUtils.GetDefaultValue(typeof(DBNull)), Is.EqualTo(DBNull.Value));
+    }
+}
```

Comment:
Important enough to nail this down

Reply:
Updated test to show that we compare with default values.